rules/framework.bzl

```diff
@@ -581,15 +589,28 @@ def _copy_swiftmodule(ctx, framework_files):
     return [
         # only add the swift module, the objc modulemap is already listed as a header,
         # and it will be discovered via the framework search path
-        swift_common.create_module(name = swiftmodule_name, swift = swift_module),
+        swift_common.create_module(
+            name = swiftmodule_name,
+            clang = None if virtualize_frameworks else swift_common.create_clang_module(
```

Comment:
Only usable in the non-vfs case because the vfs overlay sets the path to the modulemap for that other case, setting `clang` here for vfs case breaks the build